interface/usergroup/docs_validation.php

```diff
@@ -0,0 +1,87 @@
+<?php
+  require_once("../globals.php");
+  require_once("$srcdir/acl.inc");
+  require_once("$srcdir/formdata.inc.php");
+  require_once("$srcdir/options.inc.php");
+  require_once("$srcdir/htmlspecialchars.inc.php");
+  require_once("$srcdir/headers.inc.php");
+
+  $query = "SELECT id, size, docdate, storagemethod, url FROM documents";
+  $query .= " LIMIT 500";
+  $res = sqlStatement($query);
+?>
+
+<html>
+<head>
+<?php
+  call_required_libraries(array("jquery-min-3-1-1","bootstrap","fancybox"));
+  resolveFancyboxCompatibility();
+?>
+
+<title><?php echo xlt('Documents Validation'); ?></title>
+
+<!-- style tag moved into proper CSS file -->
+
+</head>
+
+<body class="body_top">
+  <center>
+    <h2><?php echo xlt('Documents Validation'); ?></h2>
+  </center>
+  <hr>
+
+  <!-- using addressbook_list styles-->
+  <div id="addressbook_list">
+  <table class="table">
+    <tr class='head'>
+      <td><?php echo xlt('Location'); ?></td>
+      <td><?php echo xlt('Size'); ?></td>
+      <td><?php echo xlt('Date added'); ?></td>
+      <td><?php echo xlt('Storage'); ?></td>
+    </tr>
+
+  <?php
+    $listEmpty = true;
+    while ($row = sqlFetchArray($res)) {
+      $listEmpty = false;
+      $fileURL = $row['url'];
+      $location = substr($fileURL, strpos($fileURL, "LibreEHR"));
+      $size = ((float)$row['size'])/(10**6) . " MB";
```

Comment:
Is the ** here correct?

Reply:
Does that equate to MB?  Ten to the 6th power.  MiB is 1024*1024.  Never seen that style of notation/calculation before.  I guess we could like... _test the code_ to validate!

Reply:
I got a php error on it that is why I asked.

Reply:
Good reason to ask.
The answer is "no" that doesn't work :)

Reply:
Is it the math, or a division by zero issue?  Could be either if the row does not return anything.  Good idea to check if a value exists before dividing.  Also, formatting to megabytes isn't the best option for file size checks that should be <1MB

Reply:
Silly mistake by me! I will replace (10**6) with (1024*1024), which seems much better.